radio/test_sessions/tasklist_features.py

```diff
@@ -0,0 +1,11 @@
+tasks = [
+    [tc.SendBeacon(), SendLoop(2), WaitMode.NoWait],
+    [tc.SendBeacon(), SendReceive, WaitMode.NoWait],
+    [tc.SendBeacon(), Print("It shows how the Print() action works. It does not send any telecommand. It just prints a message."), WaitMode.NoWait],

```

Comment:
Using print in that way is a nightmare
![Scream](https://media.giphy.com/media/KupdfnqWwV7J6/giphy.gif)

Reply:
Tak, wiem. @mandrek44 na to samo zwrócił uwagę :)

Reply:
Fxied :white_check_mark: